lib/server/db/mysql/migrator.go

```diff
@@ -103,6 +103,10 @@ func (m *Migrator) MigrateCertificatesTable() error {
 		if err != nil {
 			return err
 		}
+		_, err = tx.Exec(funcName, "ALTER TABLE certificates MODIFY pem varbinary(8192)")
+		if err != nil {
+			return err
+		}
```

Comment:
And this does not look like it would apply if the first migration took place, right?

Reply:
good catch